app/core/config.py

```diff
@@ -157,10 +150,29 @@ class Settings(BaseSettings):
     # NOTE: AUTH_CLIENTS property should never be used in the code. To get an auth client, use `KNOWN_AUTH_CLIENTS`
     AUTH_CLIENTS: list[tuple[str, str | None, list[str], str]]
 
+    #################################
+    # Hardcoded Hyperion parameters #
+    #################################
+
+    # Hyperion follows Semantic Versioning
+    # https://semver.org/
+    HYPERION_VERSION: str = "2.6.0"
+    MINIMAL_TITAN_VERSION_CODE: int = 113
+
     ######################################
     # Automatically generated parameters #
     ######################################
 
+    # If Hyperion should initialize the database on startup
+    # This environment variable is set by the Gunicorn on_starting hook, to tell the workers to avoid initializing the database
+    # You don't want to set this variable manually
+    HYPERION_INIT_DB: bool = True
+
+    # An old variable used by clients to determine if they are compatible with Hyperion
+    # Using 0.0.1 as the minimal version, force old and broken clients to show a message asking to update
+    # TODO: remove this property when old clients are updated
+    MINIMAL_TITAN_VERSION: str = "0.0.1"  # deprecated, use MINIMAL_TITAN_VERSION_CODE
```

Comment:
Maybe it is time to remove this property ?

If we keep it, why did you put it in automatically generated parameters ?

Reply:
Good idea, I will make a second PR